cheat.php

```diff
@@ -128,9 +128,9 @@
 	$LagAdjustedWaitTime = $WaitTime - $SkippedLagTime;
 	$PlanetCheckTime = microtime( true );
 
-	Msg( '   {grey}Waiting 60 seconds before rescanning planets...' );
+	Msg( '   {grey}Waiting 100 seconds before rescanning planets...' );
 
-	sleep( 60 );
+	sleep( 100 );
```

Comment:
This is assuming that all requests can finish in 9 seconds, which they do not always do. Instead I would try adjusting pace calculation by time delta between checks and see if that improves anything.

Reply:
Alright, I'll get on that